proposals/utils/proposal_utils.py

```diff
@@ -97,15 +97,21 @@ def available_urls(proposal):
         urls.append(studies_url)
 
 
-        consent_url = AvailableURL(title=_('Documenten'))
+        consent_docs_url = AvailableURL(title=_('Uploaden'), url=reverse('proposals:consent', args=(proposal.pk, )))
+        translated_docs_url = AvailableURL(title=_('Vertaling'), url=reverse('proposals:translated', args=(proposal.pk, )))
```

Comment:
Some of these lines are very long. Wrapping makes them hard to read and they hide the structure of the function call. Compare:

    translated_docs_url = AvailableURL(title=_('Vertaling'), url=reverse('proposals:translated', args=(proposal.pk, )))
    consent_url = AvailableURL(title=_('Formulieren'), children=[translated_docs_url, consent_docs_url])

With the following:

    translated_docs_url = AvailableURL(
        title=_('Vertaling'),
        url=reverse(
            'proposals:translated',
            args=(proposal.pk,),
        )
    )
    consent_url = AvailableURL(
        title=_('Formulieren'),
        children=[
            translated_docs_url,
            consent_docs_url
        ]
    )

I much prefer it when similar code has a similar "shape"!

I know much of the Ethics doesn't follow these guidelines very well, but we try to write all new code in line with PEP 8. If you have a small font size or wide monitor, consider turning on an IDE warning when lines get too long.